noxfile.py

```diff
@@ -68,6 +69,15 @@ def run_coverage(session):
         if sys.version_info > (3, 12):
             session.install("-e", ".[all,jax]", silent=False)
         else:
+            session.run_always(
+                sys.executable,
+                "-m",
+                "pip",
+                "cache",
+                "remove",
+                "scikits.odes",
+                external=True,
+            )
```

Comment:
If a cached wheel is present, `PIP_NO_BINARY` does not apply and `pip` prefers using it. This command removes cached wheels for `scikits.odes` as necessary and exits with a warning if they are not present.